skimage/measure/simple_metrics.py

```diff
@@ -142,3 +145,48 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        eps: float = 1e-6,
+                        size: int = 3) -> float:
+    """ The image enhancement measure called EME based on [1]_.
+
+        Parameters
+        ----------
+        image : array
+            Input image of which the quality should be assessed.
+            Can be either 3-channel RGB or 1-channel grayscale.
+        size : int
+            Size of the window.
+        eps : float
```

Comment:
Do you need ``eps`` as a parameter or could you choose a value inside the function once and for all?

Reply:
I think it is better to let the user control the `eps` parameter because if the potential image/matrix is sparse. A larger `eps` could help to avoid unnecessary negative values.